vulcanexus_base/environment/vulcanexus_base-setup.bash

```diff
@@ -0,0 +1,17 @@
+# Copyright 2022 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
+export VULCANEXUS_DISTRO=galactic
```

Comment:
We could add a `VULCANEXUS_HOME` var pointing to the root of the install directory. The way to do it would be to have this file as a `.in` file which is configured using CMake to set the proper installation path regardless of things like `--merge-install`